google/cloud/aiplatform/metadata/constants.py

```diff
@@ -26,3 +26,8 @@
     SYSTEM_EXPERIMENT: _DEFAULT_SCHEMA_VERSION,
     SYSTEM_METRICS: _DEFAULT_SCHEMA_VERSION,
 }
+
+# GCP UI currently only displays experiment context which has a "experiment_deleted" key and False value.
+# This is a temporary solution to integrate tensorboard results into GCP UI.
```

Comment:
I would just mention something that this is needed until the backend supports deletion of contexts. Don't give much details about GCP UI or anything, as this is public stuff

Reply:
Good point, comment updated. Actually do we have a bug for this which we can use as a TODO here?

Reply:
b/182292666, but I don't know if we should be linking internal bugs either

Reply:
Hmm, I did saw there was some bug numbers in TODOs in other files. I just created a TODO w/o the bug number just in case.